app/routes/players.tsx

```diff
@@ -39,16 +39,16 @@ export default function NotesPage() {
             type="submit"
             className="rounded bg-slate-600 py-2 px-4 text-blue-100 hover:bg-blue-500 active:bg-blue-600"
           >
-            Logout
+            Iniciar Sesión 
```

Comment:
Creó que aquí quieres decir:
```suggestion
            Cerrar sesión 
```